File: AppGenerateFiles/security/Model/Tbl_Profile.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Tbl_Profile : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Perfil { get; set; }
       public string? Nombres { get; set; }
       public string? Apellidos { get; set; }
       public DateTime? FechaNac { get; set; }
       public int? IdUser { get; set; }
       public string? Sexo { get; set; }
       public string? Foto { get; set; }
       public string? DNI { get; set; }
       public string? Correo_institucional { get; set; }
       public int? Id_Pais_Origen { get; set; }
       public string? Estado { get; set; }
   }
}

File: UI/wwwroot/WDevCore/Security/Tbl_Profile.js
//@ts-check
//import { Cat_Dependencias, Tbl_Servicios } from "../../ModelProyect/Tbl_CaseModule.js";
import { WForm } from "../WComponents/WForm.js";
//import { FilterData } from "../WModules/CommonModel.js";
import { EntityClass } from "../WModules/EntityClass.js";
//import { type } from "../WModules/WComponentsTools.js";

//@ts-check
class Tbl_Profile extends EntityClass {
    constructor(props) {
        super(props, 'EntityHelpdesk');
        for (const prop in props) {
            this[prop] = props[prop];
        }
    }
    Id_Perfil = { type: 'number', primary: true };
    Nombres = { type: 'text' };
    Apellidos = { type: 'text' };
    FechaNac = { type: 'date' };
    Sexo = { type: "Select", Dataset: ["Masculino", "Femenino"] };
    Foto = { type: 'img' };
    DNI = { type: 'text' };
    Correo_institucional = { type: 'text', label: "correo", disabled: true, hidden: true };
    Estado = { type: "Select", Dataset: ["ACTIVO", "INACTIVO"] };    
}
export { Tbl_Profile }